pyrs/transpiler.py

```diff
@@ -333,6 +333,10 @@ def visit_IntEnum(self, node):
         fields = "\n".join(fields)
         return f"enum {node.name} {{\n{fields}\n}}\n\n"
 
+    def visit_StrEnum(self, node):
+        self._usings.add("strum_macros::EnumString")
+        return f"#[derive(Debug, PartialEq, EnumString)]\n{self.visit_IntEnum(node)}"
```

Comment:
Rename logic to `visit_Enum` or `visit_CommonEnum` and then have Int/StrEnum reuse the same code.
Also a test case similar to int_enum.py for code coverage in the same commit.